File: src/com/schoolke/bean/Notice.java
package com.schoolke.bean;

/**
 * Created by Bragg Troy on 2017/3/20.
 */
public class Notice {
    private int id;
    private String name;
    private String time;
    private String text;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public Notice(int id, String name, String time, String text) {
        this.id = id;
        this.name = name;
        this.time = time;
        this.text = text;
    }
    public Notice(){}
}

File: web/js/notice.js
/**
 * Created by Bragg Troy on 2017/3/15.
 */
getHeader();
getNav();

$(function () {
    var noticeId = searchString("noticeId").noticeId || 0;
    ajax_get("notice",{"ID":noticeId},function(res){
        res = JSON.parse(res);
        document.title = "公告 - " +　res.name;
        $('.title').html(res.name);
        $('.time').html("时间："+ res.time);
        $('.text-content').html(res.text);
    })

    // 加载公告列表


})

getFooter();
